custom/ilsgateway/tanzania/warehouse/updater.py

```diff
@@ -331,9 +244,13 @@ def populate_report_data(start_date, end_date, domain, runner):
         res.get()
 
     non_facilities_chunked_list = chunked(non_facilities, 50)
+
     # then populate everything above a facility off a warehouse table
     for chunk in non_facilities_chunked_list:
-        res = chain(process_non_facility_warehouse_data.si(org, start_date, end_date, runner) for org in chunk)()
+        res = chain(
+            process_non_facility_warehouse_data.si(org, start_date, end_date, runner, strict)
+            for org in chunk
+        )()
```

Comment:
I'm not sure - it might be because of the memory issues pointed out by @kkrampa here: https://github.com/dimagi/commcare-hq/commit/598e00828821b7123cb10ab39cf24f93d7d3edd4#commitcomment-10855444 or it might just be accidental (this code was just moved around)